content/docs/rfcs/12/README.md

```diff
@@ -61,77 +63,151 @@ The following are not considered as part of the adversarial model:
 ## Protobuf
 
 ```protobuf
-message FilterRequest {
-  bool subscribe = 1;
-  string topic = 2;
-  repeated ContentFilter contentFilters = 3;
-
-  message ContentFilter {
-    string contentTopic = 1;
+syntax = "proto3";
+
+// 12/WAKU2-FILTER rfc: https://rfc.vac.dev/spec/12/
+package waku.filter.v2;
+
+// Protocol identifier: /vac/waku/filter-subscribe/2.0.0-beta1
+message FilterSubscribeRequest {
+  enum FilterSubscribeType {
+    SUBSCRIBER_PING = 0;
```

Comment:
Mmm. Perhaps, but the idea is here to simply see if the subscriber (not a specific subscription) has any active subscription. It's meant to be as lightweight as possible so it can be performed with some regularity (simply returns a "200-OK" or error). A future fault tolerant filter protocol could perhaps return the full subscription details to a client with `get_subscriptions`, but I think that would be too complex/bandwidth-heavy for the very basic functionality I envisioned here.